lib/components/base/Pagination.vue

```diff
@@ -26,7 +26,7 @@
         v-else
         :class="{
           'page-number current': page === item,
-          shrink: item > 99,
+          shrink: (item as number) > 99,
```

Comment:
```suggestion
          shrink: item > 99,
```

Once again, assuming you narrowed the type of `pages` ref, this cast becomes unnecessary because TypeScript will be able to narrow this type to `number` based on the `v-else` (just a few lines above there's a `v-if` that checks that item is a `'-'`, so this means `item` here can only be a `number`).